pgml-sdks/pgml/javascript/tests/typescript-tests/test.ts

```diff
@@ -92,29 +92,30 @@ it("can search", async () => {
       query: {
         full_text_search: { body: { query: "Test", boost: 1.2 } },
         semantic_search: {
-          title: { query: "This is a test", boost: 2.0 },
```

Comment:
Not for this PR, but in the future it might be nice to have a "needle" document, and use it's ID, so you don't have unstable result IDs because all the documents are equally similar "This is a test document".